docs(schema-review): add features/schema-review docs (#234)
bytebase/bytebase.com: content/docs/en/use-bytebase/schema-review/overview.md

---
title: Schema Review
---
<hint-block type="warning">

> Schema Review requires **Team Plan**.

Reviewer: ```suggestion
Schema Review requires at least **Team Plan**.
```

</hint-block>

This document is to show how to configure schema review.

<hint-block type="info">

> You can know what is schema review [here](/docs/features/schema-review/overview).

Reviewer: THe link can be inlined with the configure `schema review` above

</hint-block>

## How Schema Review Works

> Bytebase defines `Schema Review Policy` for each `Environment`. The `Schema Review Policy` is essentially a set of SQL lint rules, and we call SQL lint rule the `Schema Review Rule`. After this, Bytebase will check SQL according to the corresponding `Schema Review Policy`.

Reviewer: ```suggestion
Bytebase defines `Schema Review Policy` for each `Environment`. The `Schema Review Policy` is essentially a set of SQL lint rules, and we call SQL lint rule the `Schema Review Rule`. Once configured, Bytebase will check SQL according to the corresponding `Schema Review Policy`.
```
